Neos.Flow/Classes/Mvc/Exception/ForwardException.php

```diff
@@ -19,32 +19,21 @@
  *
  * @api
  */
-class ForwardException extends StopActionException
+final class ForwardException extends \Neos\Flow\Mvc\Exception
 {
     /**
-     * @var ActionRequest
+     * @var ActionRequest The next request, containing the information about the next action to execute.
      */
-    protected $nextRequest;
+    public readonly ActionRequest $nextRequest;
 
-    /**
-     * Sets the next request, containing the information about the next action to
-     * execute.
-     *
-     * @param ActionRequest $nextRequest
-     * @return void
-     */
-    public function setNextRequest(ActionRequest $nextRequest)
+    private function __construct(string $message, int $code, ?\Throwable $previous, ActionRequest $nextRequest)
     {
+        parent::__construct($message, $code, $previous);
         $this->nextRequest = $nextRequest;
     }
 
-    /**
-     * Returns the next request
-     *
-     * @return ActionRequest
-     */
-    public function getNextRequest()
+    public static function create(ActionRequest $nextRequest)
```

Comment:
then maybe same signature here? with request and exceptionmessage?

Reply:
Yes added a $details param for the ForwardException as well.